kkarsnia/BarFinder-ATX
# fixed the favorited venue icon positioning in row item

**Why Did I Do That?**
so it will not get cutoff or impede the distance textview any longer.

**Steps to Test**
-Run app
-Click the COOL, SHOW BARS NEAR ME button
-Observe the favorited icon is now showing without impeding the distance textview or getting it's own imageview cut off at the top, it is now resizing and aligned properly with the distance view (start of/top)**

**note that if you have not favorited a venue before, you will not see the favorited icon in the barpicker view. To favorite a venue and see this show in barpicker view, select a venue from the barpicker list and click the star in the center of the screen between the map and venue details. Then repeat the above steps to test.

**Any Known Risks?**
The image does resize significantly smaller so it may run up against accessibility issues (think: poor eyesight, riding down the steep slope of the Ballmer Peak, etc.)
